README.md

```diff
@@ -1,2 +1,45 @@
 # terraform-aws-s3-cloudfront-site
-A terraform module to create an S3 Website behind a Cloudfront distribution
+A terraform module to create an S3 Website. This module creates an S3 bucket with read-only access for anonymous users and a logs bucket. 
+
+# Usage
+```hcl
+module "static_site" {
+  source                      = "github.com/azavea/terraform-aws-s3-site"
+  static_site_bucket_name     = "mysite-bucket"
+  static_site_log_bucket_name = "mysite-logs-bucket"
+  project                     = "Unknown"
+  environment                 = "Unknown"
+}
```

Comment:
I think that we should include an additional parameter for `region`, in case we need to create the bucket outside of `us-east-1`.